website/src/pages/news.tsx

```diff
@@ -0,0 +1,103 @@
+import ArrowRightIcon from 'mdi-react/ArrowRightIcon'
+import * as React from 'react'
+import { Helmet } from 'react-helmet'
+import { ContentSection } from '../components/content/ContentSection'
+import Layout from '../components/Layout'
+export default class NewsPage extends React.Component<any, any> {
+    constructor(props: any) {
+        super(props)
+        this.state = {
+            events: [],
+        }
+    }
+
+    public componentDidMount(): void {
+        if (document) {
+            document.getElementsByTagName('body')[0].setAttribute('style', 'background-image:none')
+        }
+    }
+
+    public render(): JSX.Element | null {
+        const desc =
+            'Sourcegraph in the news.'
+        return (
+            <Layout location={this.props.location}>
```

Comment:
A more concise way to set meta attributes is passing a dict to the Layout component - https://github.com/sourcegraph/about/blob/master/website/src/pages/case-studies/lyft-monolith-to-microservices.tsx#L8